code/modules/surgery/tools.dm

```diff
@@ -279,78 +241,61 @@
 
 /obj/item/scalpel/advanced
 	name = "laser scalpel"
-	desc = "An advanced scalpel which uses laser technology to cut. It's set to scalpel mode."
+	desc = "An advanced scalpel which uses laser technology to cut."
 	icon = 'icons/obj/surgery.dmi'
 	icon_state = "scalpel_a"
 	hitsound = 'sound/weapons/blade1.ogg'
 	force = 16
 	toolspeed = 0.7
 	light_color = LIGHT_COLOR_GREEN
+	sharpness = IS_SHARP_ACCURATE
 
 /obj/item/scalpel/advanced/Initialize()
 	. = ..()
 	set_light(1)
 
 /obj/item/scalpel/advanced/attack_self(mob/user)
-	playsound(get_turf(user),'sound/machines/click.ogg',50,TRUE)
-	var/obj/item/circular_saw/advanced/saw = new /obj/item/circular_saw/advanced(drop_location())
-	to_chat(user, "<span class='notice'>You incease the power, now it can cut bones.</span>")
-	qdel(src)
-	user.put_in_active_hand(saw)
-
-/obj/item/circular_saw/advanced
-	name = "laser scalpel"
-	desc = "An advanced scalpel which uses laser technology to cut. It's set to saw mode."
-	icon = 'icons/obj/surgery.dmi'
-	icon_state = "saw_a"
-	hitsound = 'sound/weapons/blade1.ogg'
-	force = 17
-	toolspeed = 0.7
-	sharpness = IS_SHARP_ACCURATE
-	light_color = LIGHT_COLOR_GREEN
-
-/obj/item/circular_saw/advanced/Initialize()
-	. = ..()
-	set_light(2)
+	playsound(get_turf(user), 'sound/machines/click.ogg', 50, TRUE)
+	if(tool_behaviour == TOOL_SCALPEL)
+		tool_behaviour = TOOL_SAW
+		to_chat(user, "<span class='notice'>You incease the power of [src], now it can cut bones.</span>")

```

Comment:
Typo, increase, not incease, perhaps rewritting the word might help with legibility as well.

Reply:
ertgasdha